weaviate/collection/classes/internal.py

```diff
@@ -36,6 +37,12 @@ class _Object(Generic[P]):
     metadata: _MetadataReturn
 
 
+@dataclass
+class _GenerativeReturn(Generic[P]):
+    objects: List[_Object[P]]
+    generative_combined_result: Optional[str] = None
```

Comment:
So, to be clear here, `_GenerativeReturn.generative_combined_result` will return the LLM summary of the objects grouped together and `_MetadataReturn.generative` will return the LLM summary of the specific object?

Can users make a generative query and fail to specify `MetadataQuery(generative=True)` and so not retrieve back their LLM generations?  If so, is this something we should default for them?

Reply:
see below, `MetadataQuery(generative=True) ` does not exist and cannot be forgotten - the reply is send automatically if a generative query is present